src/test/datascience/notebook/ipywidget.vscode.common.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
did you misname this file? .vscode.test.ts?

Reply:
nevermind, I see it imported. Does it need the .common extension though?

Reply:
Yeah otherwise mocha includes it.